packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/classes_table_diff.dart

```diff
@@ -79,15 +79,15 @@ class DiffClassNameColumn extends ColumnData<DiffClassStats>
       mainAxisAlignment: MainAxisAlignment.spaceBetween,
       children: [
         Expanded(child: defaultHeaderRenderer()),
-        classFilterButton,
+        controller.classFilterButton,
```

Comment:
create the ClassFilterButton here directly so that the widget is created during the widget lifecycle (since buildHeader is called during the build of the Table)